### PR TITLE
MINOR: Fix wrong Import control declaration

### PR DESCRIPTION
**Pull request by kamalcph**
### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by kamalcph** [comment]
@chia7712 @mumrah 
Please take a look.

**Comment by kamalcph** [comment]
@mumrah @chia7712 
Ping for review.

**Comment by kamalcph** [comment]
@mumrah ping for review.

File: checkstyle/import-control-core.xml [hunk 1]
```diff
@@ -65,7 +65,7 @@
     <allow pkg="kafka.api"/>
     <allow pkg="kafka.server"/>
     <allow pkg="org.apache.kafka.clients.admin"/>
-    <allow pkg="integration.kafka.server" class="IntegrationTestHelper"/>
```
Comment (dengziming) [inline checkstyle/import-control-core.xml:83]: pkg has been changed to kafka.server

**Review by dengziming — COMMENTED** [review]
A minor comment

File: checkstyle/import-control-core.xml (hunk 1 above)
Comment (kamalcph) [reply checkstyle/import-control-core.xml:83]: Updated. Also, closed #10099 as it's done.

**Pull request closed without merging**